### PR TITLE
Fix/3696  servdev

### PR DESCRIPTION
**Pull request by grignards**
See #3696.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

## Testing method

- login/out and verify the TLMs are properly transmitted
- launch applications in servdev mode, for example app file:
  - add `http://localhost:8075` to `development.ini/cors.access-control-allowed-origin`
  - run backend: `docker-compose up -d && pserve development.ini`
  - run `yarn run servdev`
  - go to http://localhost:8075
- check servdev apps are usable.

### CODE REVIEW

File: frontend/dist/index.html [hunk 1]
```diff
@@ -1,55 +1,116 @@
 <!DOCTYPE html>
 <html>
+
```
Comment (raphj) [inline frontend/dist/index.html:3]:
```suggestion
```

**Review by raphj — APPROVED** [review]

File: frontend/test/setup.js [hunk 2]
```diff
@@ -4,8 +4,6 @@ import Enzyme from 'enzyme'
 import chaiEnzyme from 'chai-enzyme'
 import Adapter from 'enzyme-adapter-react-16'
 import sinon from 'sinon'
-import EventSource from 'eventsourcemock'
-import AbortController from 'abort-controller'
```
Comment (Skylsmoi) [inline frontend/test/setup.js:8]: I think these dependencies should be removed from forntend/package.json

File: frontend_lib/test/setup.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 import { JSDOM } from 'jsdom'
 import chai from 'chai'
+import sinon from 'sinon'
 import Enzyme from 'enzyme'
 import chaiEnzyme from 'chai-enzyme'
 import Adapter from 'enzyme-adapter-react-16'
+import EventSource from 'eventsourcemock'
+import AbortController from 'abort-controller'
```
Comment (Skylsmoi) [inline frontend_lib/test/setup.js:8]: I think these dependencies should be added to frontend_lib/package.json

**Review by Skylsmoi — CHANGES_REQUESTED** [review]

File: frontend/test/setup.js (hunk 2 above)
Comment (grignards) [reply frontend/test/setup.js:8]: Done, also moved them to dev dependencies as they are used only in tests.

File: frontend_lib/test/setup.js (hunk 3 above)
Comment (grignards) [reply frontend_lib/test/setup.js:8]: See comment above.

**Review by Skylsmoi — APPROVED** [review]

**Pull request merged**